src/test/java/dev/openfeature/sdk/MutableTrackingEventDetailsTest.java

```diff
@@ -0,0 +1,49 @@
+package dev.openfeature.sdk;
+
+import com.google.common.collect.Lists;
+import org.junit.jupiter.api.Test;
+
+import java.time.Instant;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class MutableTrackingEventDetailsTest {
+
+
+    @Test
+    void hasDefaultValue() {
+        MutableTrackingEventDetails track = new MutableTrackingEventDetails();
+        assertEquals(0.0f, track.getValue());
```

Comment:
Shouldn't the default value be `null` per the spec?

Reply:
Ya, as @lukas-reining pointed out, we should consider this point carefully: https://github.com/open-feature/java-sdk/pull/1228#discussion_r1870292488